libs/services/designer-client-services/src/lib/base/operationmanifest.ts

```diff
@@ -411,6 +411,12 @@ export function getBuiltInOperationInfo(definition: any, isTrigger: boolean): Op
         operationId: isTrigger ? appservicetrigger : appservice,
       };
 
+    case function_:
```

Comment:
Revert changes in this file, if there is kind or any other complicated check for connectorid and operationId it is in this switch case else it is in the dictionary below